project/Dependencies.scala

```diff
@@ -96,7 +95,6 @@ object Dependencies {
     scalaLogging,
     swaggerUi,
     ficus,
-    cats,
```

Comment:
is this getting pulled in through some other dep now?

Reply:
yea..`workbench-utils` pulls this in transitively now